.github/workflows/pr_verification.yml

```diff
@@ -386,6 +386,8 @@ jobs:
 
       - name: Run Spec tests
         uses: gradle/gradle-build-action@v2
+        env:
+          MYSQL_IN_USE: ${{ matrix.database == 'mariadb' && 'true' || 'false' }}
```

Comment:
I'm a bit confused: what is the purpose of the `&& 'true' || 'false'` ? wouldn't `${{ matrix.database == 'mariadb' }}` be enough?

Reply:
Yeah, that is definitely worth trying!

Reply:
It works fine with the latter. Changed.